src/components/RegistrationForm/RegistrationForm.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from "react";
+function RegistrationForm(props) {
+  const [state, setState] = useState({
```

Comment:
can we rename this to something more meaningful to this component? like formValues, formState or something

Reply:
You mean instead of useState?